File: WebLeave/API/Dtos/Report/ReportShow/ExportExcelGridDto.cs
namespace API.Dtos.Report.ReportShow
{
    public class ExportExcelGridDto
    {
        public string LeaveDate { get; set; }
        public string SEAMP { get; set; }
        public string Applied { get; set; }
        public string Approved { get; set; }
        public string Actual { get; set; }
        public string MPPoolOut { get; set; }
        public string MPPoolIn { get; set; }
        public string Total { get; set; }
        public ReportIndexViewModelDTO ReportIndexViewModelDTO { get; set; }

    }


}

File: WebLeave/SPA/src/app/_core/models/report/report-show/export-excel-grid-dto.ts
import { ReportIndexViewModel } from './report-index-view-model.model';

export interface ExportExcelGridDto {
  leaveDate: string;
  sEAMP: string;
  applied: string;
  approved: string;
  actual: string;
  mPPoolOut: string;
  mPPoolIn: string;
  total: string;
  reportIndexViewModelDTO: ReportIndexViewModel;
}
